Makefile

```diff
@@ -171,7 +171,7 @@ gen-proto:
 .PHONY: gen-flat
 gen-flat:
 	# -o /pkg generates into same folder as tempo.fbs for simpler imports.
-	docker run -v${PWD}:/opt/src neomantra/flatbuffers flatc --go -o /opt/src/pkg /opt/src/pkg/tempofb/tempo.fbs
+	docker run --rm -v${PWD}:/opt/src neomantra/flatbuffers flatc --go -o /opt/src/pkg /opt/src/pkg/tempofb/tempo.fbs
```

Comment:
Unrelated to this PR, but I had some orphaned containers on local so added this flag.